mwls/eternal_points_list_1_0.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
The only reason I'm apprehensive about this is that some entries belong to multiple formats; namely Snapshot having the Standard Banlist that was active at the end of FFG's time.

Reply:
They're now organised into directories based on their (main) format so that field is implicitly there.

Reply:
is snapshot the only case we have of that?  If so, it seems like making the associations explicit at the cost of 1 duplicate ban list for 1 format seems like a fine tradeoff.

Reply:
(Oops, missed this reply) I think it is, and I doubt any other official format will share a banlist so yeah that's probably fair enough.

Reply:
I've been supporting Snapshot+ (Snapshot with Magnum Opus) since it appears on jnet. This would require _two_ copies of the Snapshot ban list to be added. Do we want to do that or just not have representation for Snapshot+? I have no idea if anyone actually plays it.

Reply:
snapshot vs. snapshot + sound like card pools, not formats, right?  Should we handle that with 2 card pools for the 1 snapshot format?  Wdyt?

I'm also happy to just deal with the duplication and add a test that says "those should be the same".  Let me know what you prefer.

Reply:
Oh yeah, I've overcomplicated things.

Reply:
Okay that should all be done. I've chucked a sneaky extra commit to get rid of NAPD and draft from the formats directory since I forgot to do that earlier.

Reply:
don't support snapshot+ lol. it is/was a quick and dirty solution for players who wanted snapshot plus magnum opus cards but that isn't official anywhere and probably be removed.

Reply:
Fair. RIP the only format where the orgcrime cards are legal.